package.json

```diff
@@ -6,7 +6,7 @@
   },
   "name": "@scalar-labs/scalardl-javascript-sdk-base",
   "repository": "https://github.com/scalar-labs/scalardl-javascript-sdk-base",
-  "version": "3.2.1",
+  "version": "3.2.2",
```

Comment:
@supl Not directly related to this PR but, one thing I would like to discuss is the versions of non-Java SDKs.
Currently, Docker containers (scalar-ledger) and Java SDK versions are consistent but non-java SDK versions are different, which makes it a bit hard to see what versions are compatible.
Do you have any good ideas?

Reply:
@feeblefakie
As long as we keep the major versions of the node SDK package and the web SDK package consistent with Scalar DL ledger version, I guess it would be OK because the Scalar DL application developers probably wouldn't be aware of the base package.

For the part of the version of the base package (to not confuse ourself), to be honest I don't have a good solution yet.

One tricky solution in my mind is,
we intentionally publish the same code snap of latest version `3.2.1` (before this PR) to version `2.7.0`, and, we change current node SDK and web SDK to rely on base 2.7.0.

However, a trick is a trick.
it will cause a philosophy drawback that 2.7.0 is not backward compatible from 2.6.1.

What do you think?

Reply:
@supl
>  I guess it would be OK because the Scalar DL application developers probably wouldn't be aware of the base package.

Right, that's true.
Ok, let's leave it as it is.
Anyway, I think we need to carefully publish the versions of the node and web SDKs.
I think we had better disable the automatic publishing since we now use semantic versioning.

Reply:
OK.
Let me disable the CI this time too :)

Reply:
@feeblefakie 
This PR also change the Circle CI workflow.
Now it only tries to publish this package to NPM registry when a tag `v[0-9]+.[0-9]+.[0-9]+` is pushed :)

Reply:
Thank you!